### PR TITLE
Update to CorsixTH v0.68 and modules to latest versions

### PR DESCRIPTION
**Pull request by rmnvgr**

### CODE REVIEW

**Comment by TheCycoONE** [comment]
Lgtm

**Comment by rmnvgr** [comment]
bot, build

**Comment by rmnvgr** [comment]
Okay after issues with the ffmpeg repository, Flathub, music not playing, it's now ready to be merged when v0.68 officially releases.

**Comment by TheCycoONE** [comment]
Thanks @rmnvgr

File: com.corsixth.corsixth.yml [hunk 1]
```diff
@@ -86,21 +71,36 @@ modules:
...
-            path: timidity.cfg
+          - type: git
+            url: https://github.com/libsdl-org/SDL_mixer.git
+            tag: release-2.8.0
+            commit: a37e09f85d321a13dfcf0d4432827ee09beeb623
+            x-checker-data:
+              type: git
+              tag-pattern: ^release-2.([\d\.]+)$
+        modules:
+          - shared-modules/linux-audio/fluidsynth2.json
```
Comment (TheCycoONE) [inline com.corsixth.corsixth.yml:89]: Are we using fluidsynth (and typically a soundfont like FluidR3_GM) or timidity/freepats. It seems a little mixed?

Comment (rmnvgr) [reply com.corsixth.corsixth.yml:89]: We’re using Fluidsynth with Freepats, Fluidsynth loads the Timidity config but Timidity is not actually installed. I tried to use the FluidR3 soundfont but it didn’t seem to work.

Comment (TheCycoONE) [reply com.corsixth.corsixth.yml:89]: Interesting ok

**Pull request merged**